dipy/viz/plotting.py

```diff
@@ -0,0 +1,82 @@
+"""
+
+plotting functions
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
this module is optional for DIPY. 

Like `cvxpy`, you need to use `plt, _, _ = optional_package("matplotlib.pyplot")`

Reply:
Hello Serge, 

there are a number of tutorials using matplotlib. Perhaps those should also have the same statement? or why not just shipping matplotlib with Dipy? 

In any case, I updated the file!